src/containers/builder/builder.container.tsx

```diff
@@ -33,7 +39,15 @@ const mapStateToProps = (
 });
 
 const mapDispatchToProps = mappedDispatchProps<BuilderContainerDispatchProps>({
-    onAddSurveyItem: (surveyId: string, surveyItem: SurveyItem) => new AddSurveyItemAction({ surveyId, surveyItem }),
+    onAddSurveyItem: (surveyId: string, surveyItem: SurveyItem) => {
+        return new AddSurveyItemAction({ surveyId, surveyItem });
+    },
+    onRemoveSurveyItem: (surveyId: string, surveyItem: SurveyItem) => {
```

Comment:
We'd only need the `surveyItemId` here

Reply:
As in remove `surveyId` as well and loop through all the surveys and all the items?

Or pass just `surveyItemId` alongside `surveyId`